### PR TITLE
[AdminBundle] fix timepicker of publish & unpublish modals

### PR DESCRIPTION
**Pull request by dbeerten**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #2416

### CODE REVIEW

File: src/Kunstmaan/NodeBundle/Resources/views/NodeAdmin/Modals/_unpublish.html.twig [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
                         <!-- Time -->
                         <div class="form-group">
-                            <div class="js-datepicker input-group" data-format="HH:mm" data-min-date="today" data-default-date="true" data-stepping="{{ unpublish_later_stepping }}">
+                            <div class="js-datepicker input-group" data-format="HH:mm" data-min-date="today" data-default-date="true" data-stepping="{{ unpublish_later_stepping }}" data-linked-datepicker="#unpub_date">
```
Comment (acrobat) [inline src/Kunstmaan/NodeBundle/Resources/views/NodeAdmin/Modals/_unpublish.html.twig]: should the `data-min-date="today"` option on this field be removed now?

**Comment by acrobat** [comment]
Tested in a local project and works as expected!

**Comment by acrobat** [comment]
Thanks @dbeerten!

**Comment by virtualize** [comment]
@dbeerten 
Just updatet to 5.3 from latest 5.2 and this PR seems to break the admin edit form of Article Pages.
As its results in an JS error "minDate.clone() is not a function" thrown here: https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/a49db196478608bbe6d2e24b5f0d525e0dbd159d/src/Kunstmaan/AdminBundle/Resources/ui/js/_datepicker.js#L70

Adding ` data-min-date="today" data-default-date="true" ` here
https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/a49db196478608bbe6d2e24b5f0d525e0dbd159d/src/Kunstmaan/AdminBundle/Resources/views/Form/fields.html.twig#L261
and here
https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/a49db196478608bbe6d2e24b5f0d525e0dbd159d/src/Kunstmaan/AdminBundle/Resources/views/Form/fields.html.twig#L277
seem to fix it.

But I think it would be better to update the JS in question to have sensible default values.

**Comment by acrobat** [comment]
Hi @virtualize 

Thanks for your comment, can you create a separate issue for this regression? This will make it easier to track and fix the issue. Thanks!

**Comment by virtualize** [comment]
Sure. Done. See #2444

**Pull request merged**